pr-deploy/src/github.ts

```diff
@@ -84,8 +84,12 @@ export class PullRequest {
       details_url: serveUrl,
       output: {
         title: 'Success! A test site was created.',
-        summary: `You can find it here: ${serveUrl}`,
-        text: serveUrl,
```

Comment:
Can we keep this line or do you think `https://storage.googleapis.com/amp-test-websites-1` is kind of an eyesore?

Reply:
Seems repetitive but if you think we should keep it, I can put it back :))